packages/babel-types/src/clone/cloneNode.js

```diff
@@ -27,8 +27,13 @@ function cloneIfNodeOrArray(obj, deep) {
 /**
  * Create a clone of a `node` including only properties belonging to the node.
  * If the second parameter is `false`, cloneNode performs a shallow clone.
+ * If the third parameter is true, the cloned nodes exclude location properties.
  */
-export default function cloneNode<T: Object>(node: T, deep: boolean = true): T {
+export default function cloneNode<T: Object>(
+  node: T,
+  deep: boolean = true,
+  withoutLoc: boolean = false,
```

Comment:
Deep clones also need to receive the `withoutLoc` param, which is why the tests are failing.

Reply:
Yes, I will fix it!

Reply:
I added `withoutLoc` to deep clones wich means `cloned.declarations[0].id.loc` needs to be null but I am still receiving `{}` in the test, any idea why?

Reply:
This looks like is working.

Reply:
Ah yes, sorry :sweat_smile:!